Pipfile

```diff
@@ -10,7 +10,7 @@ python-dateutil = "==2.8.2"
 [dev-packages]
 flit = "==3.8.0"
 pylint = "==2.16.3"
-mypy = "==1.0.1"
+mypy = "==1.1.1"
```

Comment:
@samwelkanda would it make sense to specify a version range rather than a fixed dependency version? In a way that any minor upgrades/patch updates automatically work with the SDK?